app/services/parameters/caster.rb

```diff
@@ -0,0 +1,118 @@
+module Parameters
```

Comment:
Could we put this under the Foreman namespace too? e.g. Foreman::Parameters::Caster